### PR TITLE
feat: create renderSaleMessages

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: src/commands/renderSalesMessage.ts [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+  return new MessageAttachment(canvas.toBuffer(), "testdemo.png")
+}
+
+const command: Command = {
+  id: "testdemo",
+  command: "testdemo",
+  brief: "Check test demo",
+  category: "Profile",
+  run: async (msg) => {
+    const data = await profile.getUserProfile(msg.guildId, msg.author.id)
```
Comment (trkhoi) [inline src/commands/renderSalesMessage.ts:457]: wrap these step into one function

File: src/commands/renderSalesMessage.ts [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+    mr: 0,
+    mb: 0,
+    radius: 20,
+  }
+  await drawRectangleAvatar(ctx, bigAvatar, msg.author)
+
+  return new MessageAttachment(canvas.toBuffer(), "testdemo.png")
+}
+
+const command: Command = {
```
Comment (trkhoi) [inline src/commands/renderSalesMessage.ts:451]: remove testdemo command

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]
